sync-diff-inspector/upstream-downstream-diff.md

```diff
@@ -6,21 +6,46 @@ aliases: ['/docs/dev/sync-diff-inspector/upstream-downstream-diff/','/docs/dev/r
+
+To enable the Syncpoint feature, set the value of the TiCDC configuration item `enable-sync-point` to `true` when creating a replication task. After enabling Syncpoint, TiCDC will periodically align the upstream and downstream snapshots according to the TiCDC parameter `sync-point-interval` during the data replication process, and will save the upstream and downstream TSO correspondences in the downstream `tidb_cdc.syncpoint_v1` table.
+
+Then, you only need to configure `snapshot` in sync-diff-inspector to verify the data of the TiDB upstream-downstream clusters. The following TiCDC configuration example enables Syncpoint for a created replication task:
+
+```toml
+# Enables SyncPoint.
+enable-sync-point = true
+
+# Aligns the upstream and downstream snapshots every 5 minutes
+sync-point-interval = "5m"
+
+# Cleans up the ts-map data in the downstream tidb_cdc.syncpoint_v1 table every hour
+sync-point-retention = "1h"
+```
 
 ## Step 1: obtain `ts-map`
 
 To obtain `ts-map`, execute the following SQL statement in the downstream TiDB cluster:
 
 ```sql
-mysql> select * from tidb_binlog.checkpoint;
-+---------------------+---------------------------------------------------------------------------------------------------------+
-| clusterID           | checkPoint                                                                                              |
-+---------------------+---------------------------------------------------------------------------------------------------------+
-| 6711243465327639221 | {"commitTS":409622383615541249,"ts-map":{"primary-ts":409621863377928194,"secondary-ts":409621863377928345}} |
-+---------------------+---------------------------------------------------------------------------------------------------------+
+mysql> select * from tidb_cdc.syncpoint_v1;
++------------------+----------------+--------------------+--------------------+---------------------+
+| ticdc_cluster_id | changefeed     | primary_ts         | secondary_ts       | created_at          |
++------------------+----------------+--------------------+--------------------+---------------------+
+| default          | default_test-2 | 435953225454059520 | 435953235516456963 | 2022-09-13 08:40:15 |
++------------------+----------------+--------------------+--------------------+---------------------+
 ```
 
+The fields in the preceding `syncpoint_v1` table are described as follows:
+
+- `ticdc_cluster_id`: The ID of the TiCDC cluster in this record.
+- `changefeed`: The ID of the changefeed that in this record. Because different TiCDC clusters might have changefeeds with the same name, you need to confirm the `ts-map` inserted by a changefeed with the TiCDC cluster ID and changefeed ID.
```

Comment:
```suggestion
- `changefeed`: The ID of the changefeed in this record. Because different TiCDC clusters might have changefeeds with the same name, you need to confirm the `ts-map` inserted by a changefeed with the TiCDC cluster ID and changefeed ID.
```